py3status/modules/whatismyip.py

```diff
@@ -47,6 +52,7 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 30
+    expected = {}
```

Comment:
cool,  I'd change this default to `None` and then set it to `{}` in `post_config_hook()`

I think that's a little nicer.